app/src/main/java/com/codesoom/assignment/application/AccountService.java

```diff
@@ -0,0 +1,96 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.AccountNotFoundException;
+import com.codesoom.assignment.domain.Account;
+import com.codesoom.assignment.domain.AccountRepository;
+import com.codesoom.assignment.dto.AccountData;
+import com.github.dozermapper.core.Mapper;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+/**
+ * 회원정보에 대한 생성, 조회, 수정, 삭제 처리를 담당합니다.
+ */
+
+@Service
+@Transactional
+public class AccountService {
+    private final AccountRepository accountRepository;
+    private final Mapper mapper;
+
+    public AccountService(AccountRepository accountRepository, Mapper dozerMapper) {
+        this.accountRepository = accountRepository;
+        this.mapper = dozerMapper;
+    }
+
+    /**
+     *
+     * @return 전체 회원정보입니다.
+     */
```

Comment:
description을 적지 않으셨어요. 다음은 제가 작성해 본 것입니다. 나머지도 모두 작성해보세요.
```suggestion
    /**
     * 전체 회원정보 목록을 리턴합니다.
     * @return 회원정보 목록
     */
```